drivers/ethernet/Kconfig.nxp_imx_netc

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ETH_NXP_IMX_NETC
+	bool "NXP IMX Ethernet and Network Controller (NETC) driver"
+	default y
+	depends on (DT_HAS_NXP_IMX_NETC_PSI_ENABLED)
+	select MDIO if DT_HAS_NXP_IMX_NETC_PSI_ENABLED
```

Comment:
this `if` is redundant , since the symbol already depends on DT_HAS_NXP_IMX_NETC_PSI_ENABLED

Reply:
Will remove it. Thanks.